docs/tutorial.md

```diff
@@ -156,19 +157,19 @@ From here, we can start to add these elements against the sketch layer again.
   
 1. Create a new layer. Make sure this new layer is *below* __Ink__ layer.
 2. Check the box for __Sketch__ in the __Layers Window__ to [unhide](edit.md#hide/unhide-layers) the sketch.
-3. Select __Ellipse__ and __Line__ to create a speech bubble over the sketched speech bubble.
-5. Select __Eraser__ to erase a part of the ellipse that's connected to the lines to complete the speech bubble.
-6. Select __Text__ and modify the font in the __Tools Settings Bar__.
+3. [Select __Ellipse__](objects.md) and [select __Line__](objects.md) to create a speech bubble over the sketched speech bubble.
+5. [Select __Eraser__](remove.md#eraser) to erase a part of the ellipse that's connected to the lines to complete the speech bubble.
+6. [Select __Text__](text.md) and modify the font in the __Tool__ bar settings.

```

Comment:
keep original but add in the links and the tools settings bar is what is already named in the overview so we have to see what naming convention we prefer

Reply:
Your general overview diagram upfrontin the Pinta Workspace says Toolbar

Reply:
no the toolbar is different. please see the overview page again. the tools settings bar is something else. if you go to overview.md > Tools Menu, you'll see the Tools Settings Bar. It's not the same as the Toolbar. This is the bar that you use to adjust your tools' properties as explained in the overview